### DISCUSSION TITLE
No compatible package found for 'en_core_news_sm' (spaCy v3.5.3)

### DISCUSSION THREAD

**Discussion by gabroliveros** [explosion/spaCy#12831] [Help: Installation] [answered]
I have installed spacy 3.5.3 but when I want to download 'en_core_news_sm' it gives me the error "No compatible package found for 'en_core_news_sm' (spaCy v3.5.3)". What do you suggest?

**Comment by rmitsch** [comment, marked as answer]
Hi @gabroliveros, check out the [docs](https://spacy.io/usage/models) on which models are available. In this case you should install [`en_core_web_sm`](https://spacy.io/models/en#en_core_web_sm).
